numerics_paper/version2/numerical_chj.tex

```diff
@@ -335,28 +338,57 @@ \section{An entropy-based blending of Rusanov and Roe}
 for a related approach in the context of the Euler equations, see
 \cite{ismail2009affordable,ismail2009proposed}.
 
-It is important to recognize also that it is not correct to simply suppress
-the carbuncle phenomenon in all situations, since it is related to a physical
-instability in the Euler equations; see \cite{moschetta2001carbuncle,elling2009carbuncle}.
-A test problem was contrived in \cite{elling2009carbuncle} to illustrate this.
-In the present work, we provide a test problem that possesses a genuine instability
-that leads to carbuncles in many numerical approximations.  This is an ideal
-test for assessing numerical methods, since neither the presence of carbuncles
-nor the complete absence of instability represents the correct behavior.
+Since Roe's solver is not entropy stable, the blended solver is also
+not guaranteed to be entropy stable.  We therefore include an additional term
+that is chosen to give entropy stability, at least if the second-order correction
+fluxes are neglected.  The complete proposed scheme can be written in
+flux-differencing form 
+We propose
+a new scheme consisting of the second-order LWL method \eqref{flux-differencing-form} using
+the first-order fluxes
+\begin{align} \label{blended-flux-1}
+  \bfF_{i+1/2} = \frac{\bff(\bfq_{i+1})+\bff(\bfq_i)}{2}
+  - \frac{1}{2} \left( \theta_{i+1/2}\lambda_{i+1/2}^{\max}\mathbb{I} + (1-\theta_{i+1/2})|\hat A_{i+1/2}| +\lambda_{i+1/2}^{\min}\mathbb{I}\right)(\bfq_{i+1}-\bfq_{i})
+\end{align}
+and correction fluxes \eqref{correction-fluxes} based on the Roe waves and
+the modified speeds
+\begin{align} \label{s_p}
+    s_{i+1/2}^p=\sgn(\hat\lambda_{i+1/2}^p)\lambda_{i+1/2}^p
+\end{align}
+with
+\begin{align}\label{lambda_p}
+  \lambda_{i+1/2}^p := \theta_{i+1/2}\lambda_{i+1/2}^{\max} + (1-\theta_{i+1/2})|\hat \lambda_{i+1/2}^p| + \lambda_\iph^{\min}.
+\end{align}
+Here $\mathbb{I}\in\mathbb{R}^{3\times 3}$ is the identity matrix,
+$\lambda_{i+1/2}^{\max}$ is the
+upper bound on the wave speed used in Rusanov's Riemann solver (see \S \ref{sec:rusanov}),
+and $\hat A_{i+1/2}$ is Roe's averaged flux Jacobian (see \S \ref{sec:roe}).
+
+In the following sections we describe the properties of this solver and
+explain how $\theta_\iph$ and $\lambda^{\min}_\iph$ are chosen.
+
+
+%It is important to recognize also that it is not correct to simply suppress
+%the carbuncle phenomenon in all situations, since it is related to a physical
+%instability in the Euler equations; see \cite{moschetta2001carbuncle,elling2009carbuncle}.
+%A test problem was contrived in \cite{elling2009carbuncle} to illustrate this.
+%In the present work, we provide a test problem that possesses a genuine instability
+%that leads to carbuncles in many numerical approximations.  This is an ideal
+%test for assessing numerical methods, since neither the presence of carbuncles
+%nor the complete absence of instability represents the correct behavior.
```

Comment:
Yeah, that's a good idea.  I have put it at the end of the introduction.  It is repetitive because I wrote something similar there already, but we can go back and clean it up.